views/api/admin/users/index.js

```diff
@@ -4,6 +4,7 @@ import http from '../../http'
 const users = {
   update: async (userId, userAttributes) =>
     http.patch(apiRoutes.admin.users.show(userId), userAttributes),
+  remove: async (userId) => http.delete(apiRoutes.admin.users.show(userId)),
```

Comment:
what is our conventions for the `remove` method? Is it `remove` or something else like `delete`?